spec/sidekiq-instrument/client_middleware_spec.rb

```diff
@@ -33,5 +38,20 @@
         }.to trigger_statsd_increment('my_other_worker.enqueue')
       end
     end
+
+    context 'without optional DogStatsD client' do
+      before do
+        @tmp = Sidekiq::Instrument::Statter.dogstatsd
+        Sidekiq::Instrument::Statter.dogstatsd = nil
+      end
+
+      after do
```

Comment:
Do we need this after do block?

Reply:
Seemed to be necessary: without it the other spec files remember that we unset `dogstatsd` on `Statter`